libs/live-fns/keybase.js

```diff
@@ -0,0 +1,28 @@
+const axios = require('../axios.js')
```

Comment:
Correct me if I'm wrong, but I think `Got` is used for new services. Re: https://github.com/amio/badgen-service/commit/c9081aa8b03c66bb8f484c047758a2dd386f71f7

Reply:
I'm planning do the refactor myself :D